src/Bugsnag.js

```diff
@@ -309,7 +309,7 @@ export class Report {
     this.stacktrace = error.stack;
     this.user = {};
 
-    if (!_handledState) {
+    if (!_handledState instanceof HandledState) {
```

Comment:
You should write `!(_handledState instanceof HandledState)` because of [this reason](https://eslint.org/docs/rules/no-unsafe-negation).

Also we should have a linter on this project!